### DISCUSSION TITLE
Qian.T62 not found on input data portal

### DISCUSSION THREAD

**Discussion by serbinsh** [E3SM-Project/E3SM#7772] [E3SM model help]
Hello

I am working on developing a new scientific workflow to run small region and single-site ELM (and ELM-FATES, as well as CLM5) as part of an optimal experimental design effort.  For this, I am building containerized versions to wrap into the workflow based on older work I did years ago.  For ELM I am starting with 1.1.0 and used a very basic setup to make sure I could build and test run within the container

```
#!/bin/bash
cd /E3SM/cime/scripts
./create_newcase --case /output/f19_g16.ICLM45 --res f19_g16 --compset ICLM45 --mach docker --compiler gnu
cd /output/f19_g16.ICLM45
./xmlchange DATM_CLMNCEP_YR_END=1972
./xmlchange PIO_TYPENAME=netcdf
./xmlchange RUNDIR=${PWD}/run 
./xmlchange EXEROOT=${PWD}/bld
./xmlchange NTASKS=1 
./xmlchange DIN_LOC_ROOT_CLMFORC=/inputdata/atm/datm7
./xmlchange DIN_LOC_ROOT=/inputdata/
cd /output/f19_g16.ICLM45
./case.setup
./case.build
```

This is a CLM45 compset with f19_g16 resolution

However, it appears the older forcing needed Qian.T62.c080727 are no longer available online?   

```
  Model datm missing file domain1 = '/inputdata//atm/datm7/topo_forcing/topodata_0.9x1.25_USGS_070110_stream_c151201.nc'
FAIL: Repo 'https://web.lcrc.anl.gov/public/e3sm/inputdata/' does not have file 'https://web.lcrc.anl.gov/public/e3sm/inputdata/atm/datm7/topo_forcing/topodata_0.9x1.25_USGS_070110_stream_c151201.nc'
Reason:
Spider mode enabled. Check if remote file exists.
--2022-03-28 22:31:58--  https://web.lcrc.anl.gov/public/e3sm/inputdata/atm/datm7/topo_forcing/topodata_0.9x1.25_USGS_070110_stream_c151201.nc
Resolving web.lcrc.anl.gov (web.lcrc.anl.gov)... 140.221.70.30
Connecting to web.lcrc.anl.gov (web.lcrc.anl.gov)|140.221.70.30|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
Remote file does not exist -- broken link!!!

  Model datm missing file file1 = '/inputdata//atm/datm7/topo_forcing/topodata_0.9x1.25_USGS_070110_stream_c151201.nc'
FAIL: Repo 'https://web.lcrc.anl.gov/public/e3sm/inputdata/' does not have file 'https://web.lcrc.anl.gov/public/e3sm/inputdata/atm/datm7/topo_forcing/topodata_0.9x1.25_USGS_070110_stream_c151201.nc'
Reason:
Spider mode enabled. Check if remote file exists.
--2022-03-28 22:31:59--  https://web.lcrc.anl.gov/public/e3sm/inputdata/atm/datm7/topo_forcing/topodata_0.9x1.25_USGS_070110_stream_c151201.nc
Resolving web.lcrc.anl.gov (web.lcrc.anl.gov)... 140.221.70.30
Connecting to web.lcrc.anl.gov (web.lcrc.anl.gov)|140.221.70.30|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
Remote file does not exist -- broken link!!!
```
I likely still have these older drivers on my cluster so I can actually test there and point to that location, which should allow this to run.  I also believe these driver data are still on the NCAR SVN system...

However, given that it still seems to be a support compset in 1.1.0, I wasn't sure if these forcing data should actually still online - so i figured I would post this.  If this is a feature, not a bug, please feel free to close as I dont need to run this case, it was just a test.

Thanks! (and apologies if this ends up being a dumb question)

**Comment by serbinsh** [comment]
For testing, I will switch to IGSWCLM45

https://github.com/E3SM-Project/E3SM/blob/v1.1.0/components/clm/cime_config/config_compsets.xml

Perhaps the QIAN compsets have been deprecated?

**Comment by serbinsh** [comment]
I did also notice this missing

```
Loading input file list: 'Buildconf/mosart.input_data_list'
Checking server https://web.lcrc.anl.gov/public/e3sm/inputdata/ with protocol wget
Loading input file list: 'Buildconf/clm.input_data_list'
Model clm no file specified for finidat
Loading input file list: 'Buildconf/cpl.input_data_list'
Loading input file list: 'Buildconf/datm.input_data_list'
  Model datm missing file domain1 = '/inputdata/atm/datm7/atm_forcing.datm7.GSWP3.0.5d.v1.c170516/domain.lnd.360x720_gswp3.0v1.c170606.nc'
FAIL: Repo 'https://web.lcrc.anl.gov/public/e3sm/inputdata/' does not have file 'https://web.lcrc.anl.gov/public/e3sm/inputdata/atm/datm7/atm_forcing.datm7.GSWP3.0.5d.v1.c170516/domain.lnd.360x720_gswp3.0v1.c170606.nc'
Reason:
Spider mode enabled. Check if remote file exists.
--2022-03-29 13:15:21--  https://web.lcrc.anl.gov/public/e3sm/inputdata/atm/datm7/atm_forcing.datm7.GSWP3.0.5d.v1.c170516/domain.lnd.360x720_gswp3.0v1.c170606.nc
Resolving web.lcrc.anl.gov (web.lcrc.anl.gov)... 140.221.70.30
Connecting to web.lcrc.anl.gov (web.lcrc.anl.gov)|140.221.70.30|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
Remote file does not exist -- broken link!!!
```

For now I am pointing to a cluster with all of these files just to test the run...

**Comment by rljacob** [comment]
Older files like that are in the CESM repo
https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata

The input data checker should download from there.  Not sure why it didn't.

**Comment by serbinsh** [comment]
Thanks.  Yes it will download the data when it locates it, I think my issue is that the files it may be looking for no longer exist on the ELM or the CESM repo.  Or at least when i was hunting them down there I was unable to locate them!

**Comment by rljacob** [comment]
At least one of the files in your original post is on the CESM repo.
https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/atm/datm7/topo_forcing/topodata_0.9x1.25_USGS_070110_stream_c151201.nc
